GMG/Scenes/ChordProgress/Segment.swift

```diff
@@ -234,50 +262,40 @@ struct Segment: View {
             Button {
                 action()
             } label: {
-                ViewThatFits(in: .horizontal) {
-                    /// CASE 1: one line
-                    Text(chord.description)
-                        .font(Typography.WantedSansStd.R7)
-                        .foregroundStyle(
-                            foregroundColor
-                        )
-                        .frame(
-                            maxWidth: .infinity,
-                            maxHeight: .infinity
-                        )
-                        .background(
-                            backgroundColor,
-                            in: RoundedRectangle(cornerRadius: 12)
-                        )
+                ZStack {
+                    if isShowDescription {
+                        ViewThatFits(in: .horizontal) {
+                            /// CASE 1: one line
+                            Text(chord.description)
+                                .font(Typography.WantedSansStd.R7)
+
+                            /// CASE 2: multi line (여유공간 있는 상태)
+                            VStack(alignment: .center) {
+                                Text("\(chord.root.description)\n\(chord.quality.description)")
+                                    .multilineTextAlignment(.center)
+                            }
+                            .font(Typography.WantedSansStd.R5)
 
-                    /// CASE 2: multi line
-                    VStack(alignment: .center) {
-                        Text(chord.root.description)
-                        Text(chord.quality.description)
+                            /// CASE 3: multi line (여유공간 없는 상태)
+                            VStack(alignment: .center) {
+                                Text("\(chord.root.description)\n\(chord.quality.description)")
+                                    .multilineTextAlignment(.center)
+                            }
+                            .font(Typography.WantedSansStd.R1)
```

Comment:
생각해보니까 VStack은 필요없네요!
```suggestion
                        Text("\(chord.root.description)\n\(chord.quality.description)")
                            .font(Typography.WantedSansStd.R5)
                            .multilineTextAlignment(.center)

                        /// CASE 3: multi line (여유공간 없는 상태)
                        Text("\(chord.root.description)\n\(chord.quality.description)")
                            .font(Typography.WantedSansStd.R1)
                            .multilineTextAlignment(.center)
```